docs/cse/automation-service/automation-service-examples.md

```diff
@@ -0,0 +1,383 @@
+---
+id: automation-service-examples
+title: Automation Examples
+sidebar_label: Automation Examples
+description: See examples that show you how to create automations for different situations.   
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+{@import ../../reuse/automation-service-la-note.md}
+
+Following are examples that show you how to create automations with the Automation Service. The examples build from simple to complex, illustrating how to perform a simple automation using an out-of-the-box integration, all the way up to creating a custom integration. These examples illustrate many of the tasks you’ll perform on a regular basis when you create your own automations. 
```

Comment:
I wonder if you should use "advanced" instead of "complex" so that it matches the header (_Advanced example_...)

Reply:
Excellent idea, @kimsauce. Done.